website/pages/en/developing/unit-testing-framework.mdx

```diff
@@ -105,7 +105,7 @@ graph test path/to/file.test.ts
 ```sh
 -c, --coverage                Run the tests in coverage mode
 -d, --docker                  Run the tests in a docker container (Note: Please execute from the root folder of the subgraph)
--f  --force                   Binary: Redownloads the binary. Docker: Redownloads the Dockerfile and rebuilds the docker image.
+-f,  --force                   Binary: Redownloads the binary. Docker: Redownloads the Dockerfile and rebuilds the docker image.
```

Comment:
can you remove the space so all of the lines are aligned?

Reply:
Got it! Good catch!